src/modules/esl-a11y-group/README.md

```diff
@@ -1,20 +1,22 @@
 # [ESL](https://exadel-inc.github.io/esl/) A11yGroup
 
-Version: *1.0.0*
+Version: _1.0.0_
 
-Authors: *Julia Murashko*
+Authors: _Julia Murashko_
 
 <a name="intro"></a>
 
 ESLA11yGroup - a helper custom element that adds a11y group behavior to targets.
-ESLA11yGroup tracks keyboard to loop focus of the targeting elements (with arrow keyboard keys navigation). 
+ESLA11yGroup tracks keyboard to loop focus of the targeting elements (with arrow keyboard keys navigation).
 
 ### Attributes / Properties
 
-- `targets` - property to declare targets for the group using [TraversingQuery](https://exadel-inc.github.io/esl/utils/esl-traversing-query/) syntax. Use group parent element as a base element.
+- `targets` - property to declare targets for the group using [TraversingQuery](https://exadel-inc.github.io/esl/utils/esl-traversing-query/) syntax. Use group parent element as a base element
 - `activate-selected` - activate the element using `click` event on selection change via esl-a11y-group
+- `prevent-scroll` - prevants scroll when target receive focus
```

Comment:
mb typo `prevents` and `receives`
if we decided to use "s" endings of the verbs, maybe change activate-selected `activate` and `Use` in targets? @ala-n, what do you think?

Reply:
Yep, I basically think that's a good point, we can try to fix docs according to our current agreements in case we touching something nearby or related to our scope changes.